docs/tensorflow.md

```diff
@@ -74,6 +74,26 @@ model.fit(x_train, y_train, epochs=args.epochs, callbacks=[hook])
 model.evaluate(x_test, y_test, callbacks=[hook])
 ```
 
+### TF 2.x GradientTape example
+```python
+import smdebug.tensorflow as smd
+hook = smd.KerasHook(out_dir=args.out_dir)
+
+model = tf.keras.models.Sequential([ ... ])
+    for epoch in range(n_epochs):
+        for data, labels in dataset:
+            dataset_labels = labels
+            # wrap the tape to capture tensors
+            with hook.wrap_tape(tf.GradientTape(persistent=True)) as tape:
```

Comment:
Is there a zero code change behavior for this?

Reply:
will this work if persistent=False?

Reply:
no zero code change behavior yet. will be introduced in the next release.

by default persistent=False (all the tests handle this case). plus there is one test for persistent=True